harness/determined/common/experimental/trial.py

```diff
@@ -51,66 +51,84 @@ def _to_bindings(self) -> bindings.v1OrderBy:
 
 
 @dataclasses.dataclass
-class TrainingMetrics:
+class TrialMetrics:
     """
-    Specifies a training metric report that the trial reported.
+    Specifies a metric that the trial reported.
 
     Attributes:
         trial_id
         trial_run_id
-        steps_completed
+        total_batches
         end_time
         metrics
         batch_metrics
     """
 
     trial_id: int
     trial_run_id: int
-    steps_completed: int
+    total_batches: int
     end_time: datetime.datetime
     metrics: Dict[str, Any]
     batch_metrics: Optional[List[Dict[str, Any]]] = None
 
+    @property
+    def steps_completed(self) -> int:
+        return self.total_batches
+
+    @steps_completed.setter
+    def steps_completed(self, value: int) -> None:
```

Comment:
why are these getter/setters for steps_completed needed?

Reply:
for compatibility with old code using these two since we're moving toward total_batches. Myabe I should have had two copies of these on Training and ValidationMetrics instead.

Reply:
I did this.